vizro-mcp/src/vizro_mcp/schemas/schemas.py

```diff
@@ -0,0 +1,131 @@
+"""MCP server for Vizro-AI chart creation best practices."""
+
+from typing import Annotated, Any, Literal, Optional
+
+import vizro.models as vm
+from pydantic import AfterValidator, BaseModel, Field
+
+# Constants used in chart validation
+CUSTOM_CHART_NAME = "custom_chart"
+
+
```

Comment:
It might be nice to have docstring(s) somewhere that explains why these exist rather than taking from our schema directly. There's various reasons, I'm sure, but currently I don't know what they are.